Gems/PhysX/Code/Editor/EditorJointConfiguration.h

```diff
@@ -71,6 +71,28 @@ namespace PhysX
         float m_limitNegative = -45.0f;
     };
 
+    /// Pair (linear) limits for joints.
+    class EditorJointLimitLinearPairConfig : public EditorJointLimitBase
+    {
+    public:
+        static const float s_linearLimitMin;
+        static const float s_linearLimitMax;
```

Comment:
nit: the naming convention for static const members don't prefix `_s`: `s_linearLimitMin` --> `LinearLimitMin`

Reply:
As there are several others like this keeping `s_` is likely okay for consistency (maybe we could change the others after) - as I think `s_` may be preferred for `static` class members, just not `static` variables/constants at file scope

Reply:
Oh yeah, that came from copy-paste from hinge joints. I fixed a couple of other instances but missed that one. Will update.